src/OSPSuite.SimModel/src/Observer.cpp

```diff
@@ -113,4 +114,19 @@ bool Observer::IsConstantDuringCalculation()
 	return IsConstant(forCurrentRunOnly);
 }
 
+bool Observer::IsUsedInFormulas() const
+{
+	return _isUsedInFormulas;
+}
+
+void Observer::SetIsUsedInFormulas(bool usedInFormulas)
+{
+	_isUsedInFormulas = usedInFormulas;
+}
+
+bool Observer::IsUsedInSimulation() const
+{
+	return _isPersistable || _isChangedBySwitch || _isUsedInFormulas;
+}
```

Comment:
That's the current definition of "Observer is used in the simulation". It is if at least one condition is true
* Observer is persistable
* Observer is changed by any switch
* Observer is used in any formula.

Reply:
I don't understand the concept of persistable for an observer. What does that mean?

Reply:
It means that it's defined as output by user. If PK-Sim or MoBi needs to display any observer, it's always set as persistable.